packages/features/ee/video/components/TeamUgradeRecordingBanner.tsx

```diff
@@ -0,0 +1,30 @@
+import { useRouter } from "next/router";
+
+import { WEBAPP_URL } from "@calcom/lib/constants";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { Icon, Button } from "@calcom/ui";
+
+export default function TeamsUpgradeRecordingBanner() {
```

Comment:
```suggestion
export default function UpgradeRecordingBanner() {
```
Also we can rename file